eng/scripts/Invoke-GenerateAndBuildV2.ps1

```diff
@@ -113,41 +113,35 @@ if ($inputFileToGen) {
 if ($relatedTypeSpecProjectFolder) {
     foreach ($typespecRelativeFolder in $relatedTypeSpecProjectFolder) {
         $typespecFolder = Resolve-Path (Join-Path $swaggerDir $typespecRelativeFolder)
-        $processScript = Resolve-Path (Join-Path $sdkPath "eng/common/scripts" "TypeSpec-Project-Process.ps1")
-        $sdkProjectFolders = Get-ChildItem -Path (Join-Path $sdkPath "sdk") -Depth 1 -Directory | Select-Object -ExpandProperty FullName
-
-        # Invoke Process script. SkipSyncAndGenerate only when it's not a new SDK project
-        $sdkProjectFolder = & $processScript $typespecFolder $commitid $repoHttpsUrl -SkipSyncAndGenerate
+        $tspConfigFile = Resolve-Path (Join-Path $typespecFolder "tspconfig.yaml")
+        $sdkProjectFolder = GetSDKProjectFolder -typespecConfigurationFile $tspConfigFile -sdkRepoRoot $sdkPath
+        $sdkAutorestConfigFile = Join-path $sdkProjectFolder "src" "autorest.md"
+        if (Test-Path -Path $sdkAutorestConfigFile) {
+            Write-Host "remove $sdkAutorestConfigFile for sdk from typespec."
+            Remove-Item -Path $sdkAutorestConfigFile
+        }
+        $repo = $repoHttpsUrl -replace "https://github.com/", ""
+        $tspclientCommand = "npx tsp-client init --tsp-config $tspConfigFile --repo $repo --commit $commitid"
```

Comment:
For SDK automation, it needs to use `--local-spec-repo` option to use local cloned spec repo as it would have issue while pulling specs from the remote private spec repo.